scripts/download-resources.mjs

```diff
@@ -167,4 +265,56 @@ export default async function main() {
     throw new Error(`Matched ${ kimSHA.length } hits, not exactly 1, for platform ${ kubePlatform } in [${ allKimSHAs }]`);
   }
   await download(kimURL, kimPath, kimSHA[0].split(/\s+/, 1)[0]);
+
+  const kuberlrVersion = '0.3.2';
+  const kuberlrBase = `https://github.com/flavio/kuberlr/releases/download/v${ kuberlrVersion }`;
+  const kuberlrBaseURL = `${ kuberlrBase }/kuberlr_${ kuberlrVersion }_${ kubePlatform }_amd64`;
+  const kuberlrPlatformDir = `kuberlr_${ kuberlrVersion }_${ kubePlatform }_amd64`;
+  const allKuberlrSHAs = await getResource(`${ kuberlrBase }/checksums.txt`);
+  const kuberlrSHA = allKuberlrSHAs.split(/\r?\n/).filter(line => line.includes(`kuberlr_${ kuberlrVersion }_${ kubePlatform }_amd64`));
+  let kuberlrPath;
+
+  switch (kuberlrSHA.length) {
+  case 0:
+    throw new Error(`Couldn't find a matching SHA for [kuberlr_${ kuberlrVersion }_${ kubePlatform }-amd64] in [${ allKuberlrSHAs }]`);
+  case 1:
+    break;
+  default:
+    throw new Error(`Matched ${ kuberlrSHA.length } hits, not exactly 1, for platform ${ kubePlatform } in [${ allKuberlrSHAs }]`);
+  }
+  const finalKuberlrSHA = kuberlrSHA[0].split(/\s+/, 1)[0];
+
+  if (onWindows) {
+    kuberlrPath = await downloadZip(`${ kuberlrBaseURL }.zip`, finalKuberlrSHA, 'kuberlr', kuberlrPlatformDir);
+  } else {
+    kuberlrPath = await downloadTarGZ(`${ kuberlrBaseURL }.tar.gz`, finalKuberlrSHA, 'kuberlr', kuberlrPlatformDir);
+  }
+
+  // Desired:
+  // copy kubectl to ~/.kuberlr/PLATFORM-amd64/kubectlMAJ.MIN.PATCH
+  // .../resources/PLATFORM/bin: symlink kubectl pointing to kuberlr
+  if (kuberlrPath) {
+    const kuberlrDir = path.join(await findHome(), '.kuberlr', `${ kubePlatform }-amd64`);
+    const pathParts = path.parse(kubectlPath);
+    const newKubectlPath = path.join(kuberlrDir, `${ pathParts.name }${ kubeVersion.replace(/^v/, '') }${ pathParts.ext }`);
+
+    try {
+      await fs.promises.access(newKubectlPath);
+      await fs.promises.rm(kubectlPath);
+    } catch (_) {
+      await fs.promises.mkdir(kuberlrDir, { recursive: true, mode: 0o755 });
+      try {
+        await fs.promises.rename(kubectlPath, newKubectlPath);
+      } catch (_) {
+        // Assume we ran into a cross-link error
+        await fs.promises.copyFile(kubectlPath, newKubectlPath);
+        await fs.promises.rm(kubectlPath);
+      }
+    }
+    if (onWindows) {
+      await fs.promises.copyFile(kuberlrPath, kubectlPath);
+    } else {
+      await fs.promises.symlink(kuberlrPath, kubectlPath);
```

Comment:
Here `kuberlrPath` is absolute; so it get dropped into the (mac) disk image as a link to the… source tree (instead of relative, to the `kuberlr` next to it).  That's probably bad.

Reply:
Done.  Now the code does the equivalent of `pushd binDir; ln -s kuberlr kubectl` -- only on macos(and linux) of course